src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -164,6 +165,11 @@ public string Caption
         public bool DefaultToCurrentUriOnRedirect { get; set; }
 
         /// <summary>
+        /// Boolean to set whether the middleware should go to user info endpoint to retrieve claims or not.
```

Comment:
We want to indicate that this is an additional step after setting the Identity to the Id_token that arrived from the TokenEndpoint.